metallb/metallb.yaml.j2

```diff
@@ -0,0 +1,434 @@
+---
```

Comment:
What consumes this file?

We should also document somewhere that installation via manifest is a temporary measure and will be removed once an operator is available.  This is just to allow easier testing of metallb sooner in an openshift environment.

Reply:
This is for manifest for metalLB with parameterize image. `config_metallb_bgp.sh` and `config_metallb_l2.sh` has variables, METALLB_IMAGE_BASE and METALLB_IMAGE_TAG so we can easily specify which metallb container image is used.
 I believe that CI case we need to do so, that is why added.

I understand that manifest must be short life until metallb operator is introduced. Once we introduce metallb operator, of course we also need to change this script as well, so it is not a big problem for now.

Reply:
I will update README to mention your comment (i.e. it is pretty temporally solution and metallb-operator will change this script)

Reply:
Update the README.